sycl/test-e2e/ESIMD/unified_memory_api/Inputs/scatter.hpp

```diff
@@ -449,3 +449,217 @@ template <typename T, TestFeatures Features> bool testSLM(queue Q) {
 
   return Passed;
 }
+
+template <typename T, uint16_t N, uint16_t VS, bool UseMask, bool UseProperties,
+          typename ScatterPropertiesT>
+bool testLACC(queue Q, uint32_t MaskStride,
+              ScatterPropertiesT ScatterProperties) {
+  constexpr uint32_t Groups = 8;
+  constexpr uint32_t Threads = 1;
+  constexpr size_t Size = Groups * Threads * N;
+  static_assert(VS > 0 && N % VS == 0,
+                "Incorrect VS parameter. N must be divisible by VS.");
+  constexpr int NOffsets = N / VS;
+  using Tuint = sycl::_V1::ext::intel::esimd::detail::uint_type_t<sizeof(T)>;
+
+  std::cout << "Local Accessor case: T=" << esimd_test::type_name<T>()
+            << ",N=" << N << ", VS=" << VS << ",UseMask=" << UseMask
+            << ",UseProperties=" << UseProperties << std::endl;
+
+  sycl::range<1> GlobalRange{Groups};
+  sycl::range<1> LocalRange{Threads};
+  sycl::nd_range<1> Range{GlobalRange * LocalRange, LocalRange};
+
+  T *Out = static_cast<T *>(sycl::malloc_shared(Size * sizeof(T), Q));
+  for (size_t i = 0; i < Size; i++)
+    Out[i] = i;
+
+  try {
+    Q.submit([&](handler &cgh) {
+       constexpr uint32_t SLMSize = N;
+       auto LocalAcc = local_accessor<T, 1>(SLMSize, cgh);
+
+       cgh.parallel_for(Range, [=](sycl::nd_item<1> ndi) SYCL_ESIMD_KERNEL {
+         ScatterPropertiesT Props{};
+         uint16_t GlobalID = ndi.get_global_id(0);
+         uint16_t LocalID = ndi.get_local_id(0);
+         uint32_t GlobalElemOffset = GlobalID * N;
+         uint32_t LocalElemOffset = LocalID * N;
+
+         simd<T, N> InVec(GlobalElemOffset, 1);
+
+         simd<uint32_t, NOffsets> ByteOffsets(0, VS * sizeof(T));
+         scatter<T, N, VS>(LocalAcc, ByteOffsets, InVec);
+         barrier();
```

Comment:
nit: if 0-th thread does not initialize all SLM and work-items read/write only to their sections of SLM, then `barrier` is not needed.